### PR TITLE
feat(images): upload images to gcp

### PR DESCRIPTION
**Pull request by MichaelSolati**

### CODE REVIEW

File: src/site/content/en/blog/fetch-metadata/index.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Fetch Metadata request headers are a new web platform security feature designed
 Requests originating from sites served by your own server (same-origin) will continue to work.
 {% endCompareCaption %}
 
-![A fetch request from https://site.example for the resource https://site.example/foo.json in JavaScript causes the browser to send the HTTP request header "Sec Fetch-Site: same-origin".](same-origin-request.png)
+{% Img src="image/admin/aRsy2xULTR4TM2sMMsbQ.png", alt="A fetch request from site.example for the resource site.example/foo.json in JavaScript causes the browser to send the HTTP request header 'Sec Fetch-Site: same-origin'.", width="800", height="176" %}
```
Comment (robdodson) [inline src/site/content/en/blog/fetch-metadata/index.md:53]: just curious if you want to leave the `https://` protocol in here. I don't think it materially changes the article but wanted to note that it is a slight tweak.

File: src/site/content/en/handbook/self-assessment-components/index.md [hunk 2]
```diff
@@ -186,7 +186,7 @@ and then include all question data _except_ the stimulus in each component objec
 
 Here's an example:
 ```yaml
-- stimulus: "![Webby](./webby.png)"
+- stimulus: "{% Img src="image/admin/Iz4mJkLrJrLk9Ntem871.png", alt="Webby", width="200", height="200" %}"
```
Comment (robdodson) [inline src/site/content/en/handbook/self-assessment-components/index.md:189]: hm this seems like a mistake maybe?

File: src/site/content/en/metrics/fcp/index.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ starts loading to when any part of the page's content is rendered on the screen.
 For this metric, "content" refers to text, images (including background images),
 `<svg>` elements, or non-white `<canvas>` elements.
 
-[![FCP timeline from google.com](fcp-filmstrip.png)](fcp-filmstrip.png)
+[{% Img src="image/admin/3UhlOxRc0j8Vc4DGd4dt.png", alt="FCP timeline from google.com](fcp-filmstrip.png)", width="800", height="311" %}
```
Comment (robdodson) [inline src/site/content/en/metrics/fcp/index.md:32]: I noticed this image has extra square brackets. It's actually a clever trick by Phil to wrap the image in an anchor so you can click on it and get the high res version (sort of like a lightbox).

I think all of the pages in /metrics/ use this technique (but I haven't seen it anywhere else).

In the short term I think we can put an `<a>` around the shortcode which links to a version of the image.
But I think this is a cool idea and maybe we should consider adding it to the shortcode at some point?

**Review by robdodson — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/fetch-metadata/index.md (hunk 1 above)
Comment (MichaelSolati) [reply src/site/content/en/blog/fetch-metadata/index.md:53]: The markdown linter doesnt like urls that arent in blocks or tags, so something in a shortcode fails the linting test. Removing the protocol lets it pass.

Comment (robdodson) [reply src/site/content/en/blog/fetch-metadata/index.md:53]: oh you can use a comment to disable that on a per file basis. I'm not sure what the rule is called but here's an example using the duplicate heading rule

```md
<!--lint disable no-duplicate-headings-->
{% Img src="image/admin/aRsy2xULTR4TM2sMMsbQ.png", alt="A fetch request from site.example for the resource site.example/foo.json in JavaScript causes the browser to send the HTTP request header 'Sec Fetch-Site: same-origin'.", width="800", height="176" %}
<!--lint enable no-duplicate-headings-->
```

File: src/site/content/en/metrics/fid/index.md [hunk 4]
```diff
@@ -99,7 +99,7 @@ something else.
 
 Consider the following timeline of a typical web page load:
 
-[![Example page load trace](fid-base.svg)](fid-base.svg)
+[{% Img src="image/admin/9tm3f6pwlHMqNKuFvaP0.svg", alt="Example page load trace](fid-base.svg)", width="800", height="260" %}
```
Comment (robdodson) [inline src/site/content/en/metrics/fid/index.md:102]: These all need linkTo

File: src/site/content/en/metrics/tbt/index.md [hunk 5]
```diff
@@ -48,7 +48,7 @@ long task that occurs between FCP and TTI.
 For example, consider the following diagram of the browser's main thread during
 page load:
 
-[![A tasks timeline on the main thread](tbt-all-tasks.svg)](tbt-all-tasks.svg)
+[{% Img src="image/admin/clHG8Yv239lXsGWD6Iu6.svg", alt="A tasks timeline on the main thread](tbt-all-tasks.svg)", width="800", height="156" %}
```
Comment (robdodson) [inline src/site/content/en/metrics/tbt/index.md:51]: linkTo

File: src/site/content/en/metrics/tti/index.md [hunk 6]
```diff
@@ -44,7 +44,7 @@ of a web page, follow these steps:
 
 The following diagram should help visualize the steps above:
 
-[![A page load timeline showing how to compute TTI](tti-timeline.svg)](tti-timeline.svg)
+[{% Img src="image/admin/WZM0n4aXah67lEyZugOT.svg", alt="A page load timeline showing how to compute TTI](tti-timeline.svg)", width="800", height="473" %}
```
Comment (robdodson) [inline src/site/content/en/metrics/tti/index.md:47]: linkTo

**Review by robdodson — APPROVED** [review]
LGTM but I found a few spots that need `linkTo`

**Pull request merged**